versioned_docs/version-5.4/02.deploying/08.publick8s/08.publick8s.md

```diff
@@ -43,6 +43,57 @@ NeuVector requires an SDN plug-in such as flannel, weave, or calico.
 
 Use the environment variable NV_PLATFORM_INFO with value platform=Kubernetes:GKE to enable NeuVector to perform GKE specific action such as running the GKE Kubernetes CIS Benchmarks.
 
+### GKE Auto Pilot support
+
+GKE Auto Pilot support is available with NeuVector from v5.4.3. Please follow below steps to deploy NeuVector on Auto Pilot cluster,
+
+An AllowlistSynchronizer should be created on the cluster before deploying the NeuVector. Here is the YAML with allowlistPath,
+
+```yaml
+apiVersion: auto.gke.io/v1
+kind: AllowlistSynchronizer
+metadata:
+  name: neuvector-allowlist
+spec:
+  allowlistPaths:
+  - SUSE/neuvector-enforcer/v1.0.0/suse-neuvector-enforcer.yaml
+  - SUSE/neuvector-scanner/v1.0.0/suse-neuvector-scanner.yaml
+```
+
+After performing "kubectl apply -f <yaml file>" check if the AllowlistSynchronizer is ready.
```

Comment:
@venkateshjayagopal 

Can we use a command to check the allowlist status? Something like `kubectl get AllowlistSynchronizer neuvector-allowlist -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}'`.  The command output should be `Ready`. I have not tried the command yet, but I assume it works. You can add it to the documentation if you’d like.